static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -160,14 +182,91 @@ define([
                     trackDisplay: true,
                     cssModifierClass: colourClass
                 }).show(renderedBanner);
+
+            if(params.secondaryButtonCaption &&  SECONDARY_BUTTON != null) {
+                setSecondaryButtonListener();
+            }
+
             if (messageShown) {
                 mediator.emit('membership-message:display');
             }
             mediator.emit('banner-message:complete');
         }
 
+        function emailIsValid(email){
+            return typeof email === 'string' && email.indexOf('@') > -1;
+        }
+
+        function sendEmail(email){
+            submitForm(email, LIST_ID).then(showThankYouMessage, showErrorMessage)
+        }
+
+        function showThankYouMessage(){
+            hideElement(REMIND_ME_TEXT_FIELD);
+            hideElement(REMIND_ME_CTA);
+            hideElement(REMIND_ME_ERROR);
+
+            showElement(REMIND_ME_THANKS_MESSAGE);
+        }
+
+        function showErrorMessage(){
+            hideElement(REMIND_ME_TEXT_FIELD);
+            hideElement(REMIND_ME_CTA);
+            showElement(REMIND_ME_ERROR);
+        }
+
+        function submitForm(email, listID) {
+            var formQueryString =
+                'email=' + encodeURI(email) + '&' +
+                'listId=' + listID;
+
+            return fetch(config.page.ajaxUrl + EMAIL_PATH,
+                        {   method: 'post',
+                            body: formQueryString,
+                            headers: {
+                                'Accept': 'application/json'
+                            }
+                        }
+            );
+        }
+
+        function getDOMElements(){
+            REMIND_ME_FORM = document.querySelector(REMIND_ME_FORM_SELECTOR);
+            REMIND_ME_TEXT_FIELD = document.querySelector(REMIND_ME_TEXT_FIELD_SELECTOR);
+            REMIND_ME_CTA = document.querySelector(REMIND_ME_CTA_SELECTOR);
+            REMIND_ME_THANKS_MESSAGE = document.querySelector(REMIND_ME_THANKS_MESSAGE_SELECTOR);
+            REMIND_ME_ERROR = document.querySelector(REMIND_ME_ERROR_SELECTOR);
+        }
+
+        function showElement(element) {
+            element.classList.remove('hide-element');
```

Comment:
Adding the class `.is-hidden` will set:

`display: none !important;`

It is included in _helpers.scss (base scss)